### PR TITLE
docs(SP): adjust the note to have subscription editions

### PR DESCRIPTION
**Pull request by vbotnari12**
listing all the editions is unnecessary especially because we are selling only Enterprise now.

### CODE REVIEW

**Comment by tbouffard** [comment]
ℹ️ the "Build PR preview" currently fails as this PR requires #2189

File: modules/ROOT/pages/connectors-execution.adoc [hunk 1]
```diff
@@ -18,7 +18,7 @@ image::images/connector_execution.png[Connector execution]
 [NOTE]
 ====
```
Comment (tbouffard) [inline modules/ROOT/pages/connectors-execution.adoc]: remove this extra line
```suggestion
```

File: modules/ROOT/pages/analytics.adoc [hunk 2]
```diff
@@ -9,7 +9,7 @@ A report presents the Administrator with data about the usage of cases and tasks
 [NOTE]
 ====
```
Comment (tbouffard) [inline modules/ROOT/pages/analytics.adoc]: remove this extra line
```suggestion
```

File: modules/ROOT/pages/customize-layouts.adoc [hunk 3]
```diff
@@ -4,7 +4,7 @@
 [NOTE]
 ====
```
Comment (tbouffard) [inline modules/ROOT/pages/customize-layouts.adoc]: remove this extra line
```suggestion
```

File: modules/ROOT/pages/enforce-password-policy.adoc [hunk 4]
```diff
@@ -6,7 +6,7 @@ A Password Policy defines a set of rules to determine whether a new password is
 [NOTE]
 ====
```
Comment (tbouffard) [inline modules/ROOT/pages/enforce-password-policy.adoc]: remove this extra line
```suggestion
```

File: modules/ROOT/pages/purge-tool.adoc [hunk 5]
```diff
@@ -15,7 +15,7 @@ The purge tool doesn't delete documents (stored in the DOCUMENT table) from the
 [NOTE]
 ====
```
Comment (tbouffard) [inline modules/ROOT/pages/purge-tool.adoc]: remove this extra line
```suggestion
```

File: modules/ROOT/pages/reporting-overview.adoc [hunk 6]
```diff
@@ -6,7 +6,7 @@ Learn how reporting can be made on application of the Bonita Platform.
 [NOTE]
 ====
```
Comment (tbouffard) [inline modules/ROOT/pages/reporting-overview.adoc]: remove this extra line
```suggestion
```

File: modules/ROOT/pages/set-up-continuous-integration.adoc [hunk 7]
```diff
@@ -6,7 +6,7 @@ Learn how to set up a continuous integration (CI) environment for your Bonita pr
 [NOTE]
 ====
```
Comment (tbouffard) [inline modules/ROOT/pages/set-up-continuous-integration.adoc]: remove this extra line
```suggestion
```

File: modules/ROOT/pages/set-up-kpis.adoc [hunk 8]
```diff
@@ -6,7 +6,7 @@ Discover how to create KPIs through the Bonita Studio feature and through a data
 [NOTE]
 ====
```
Comment (tbouffard) [inline modules/ROOT/pages/set-up-kpis.adoc]: remove this extra line
```suggestion
```

File: modules/ROOT/pages/single-sign-on-with-cas.adoc [hunk 9]
```diff
@@ -6,7 +6,7 @@ Bonita can be configured to use CAS to provide single sign-on (SSO), as long as
 [NOTE]
 ====
```
Comment (tbouffard) [inline modules/ROOT/pages/single-sign-on-with-cas.adoc]: remove this extra line
```suggestion
```

File: modules/ROOT/pages/software-extensibility.adoc [hunk 10]
```diff
@@ -3,6 +3,11 @@
 
 Learn how Bonita Platform can be extended to suit your needs.
 
+[NOTE]
+====
+For subscription editions only.
```
Comment (tbouffard) [inline modules/ROOT/pages/software-extensibility.adoc]: ❌ connectors are also available in Community

**Comment by vbotnari12** [comment]
@tbouffard there is a conflict with one file, I tried changing it, but frankly, I dont understand what's the conflict.. Can you please help? Thank you.

**Comment by tbouffard** [comment]
> @tbouffard there is a conflict with one file, I tried changing it, but frankly, I dont understand what's the conflict.. Can you please help? Thank you.

OK, I am looking at it now

**Comment by tbouffard** [comment]
✔️ merged from 2021.1 done, no more conflicts

File: modules/ROOT/pages/admin-application-overview.adoc [hunk 11]
```diff
@@ -29,15 +29,15 @@ image::images/UI2021.1/studio-get-application.png[Get application from Studio]
...
+Building the repository will create a .bos file in the `target` folder. You can import this one into the development environment. +
 Using this method is also useful if you want to get an older version of an application or if you need a weekly version with a potential bugfix.
 
 After Studio validation, the application is imported or cloned; you can view its content in the project explorer, deploy it, and open it.
 
 == Access in pre-Production and in Production
 
-To deploy the applications into a bundle or the Cloud, you can use xref:{bcdDocVersion}@bcd::bcd_cli.adoc[Bonita Continuous Delivery] (for Enterprise, Performance, Efficiency, and Teamwork editions, and using BCD).
+To deploy the applications into a bundle or the Cloud, you can use xref:{bcdDocVersion}@bcd::bcd_cli.adoc[Bonita Continuous Delivery] (for Bonita Subscription and using BCD).
+
```
Comment (tbouffard) [inline modules/ROOT/pages/admin-application-overview.adoc]:
```suggestion
```

File: modules/ROOT/pages/user-application-overview.adoc [hunk 12]
```diff
@@ -31,7 +31,8 @@ After Studio validation, the application is imported or cloned; you can view its
 
 == Access in pre-Production and in Production
 
-To deploy the applications into a bundle or the Cloud, you can use xref:{bcdDocVersion}@bcd::livingapp_manage_configuration.adoc[Bonita Continuous Delivery]'s xref:{bcdDocVersion}@bcd::bcd_cli.adoc[CLI] (for Enterprise, Performance, Efficiency, and Teamwork editions, and using BCD).
+To deploy the applications into a bundle or the Cloud, you can use xref:{bcdDocVersion}@bcd::livingapp_manage_configuration.adoc[Bonita Continuous Delivery]'s xref:{bcdDocVersion}@bcd::bcd_cli.adoc[CLI] (Subscription editions using BCD).
+
```
Comment (tbouffard) [inline modules/ROOT/pages/user-application-overview.adoc]:
```suggestion

```

File: modules/ROOT/pages/software-extensibility.adoc [hunk 13]
```diff
@@ -57,24 +54,18 @@ In the Bonita subscription edition, Bonita Studio contains xref:rest-api-extensi
...
 You can create a page using the xref:ui-designer-overview.adoc[UI Designer], which automatically creates a well-formed page.
 Note that although the page framework provided in the product is stable, we cannot guarantee that all custom pages will work with future versions, because this depends on the details of how the page is implemented.
 
 === Custom widgets
 
-In Community, Teamwork, Efficiency, Performance and Enterprise editions
-
 If the standard UI Designer xref:widgets.adoc[widgets] do not meet your needs, you can create a xref:custom-widgets.adoc[custom widget].
-You can then use your custom widget in pages, forms, and (for Subscription editions) fragments.
+You can then use your custom widget in pages, forms, and fragments - for Subscription editions.
```
Comment (tbouffard) [inline modules/ROOT/pages/software-extensibility.adoc:65]: ❓ Aren't fragments available in the Community edition in version 2021.1? or is it only available starting from a version later?
If this applies to a later version, the documentation of the related version must be updated once this PR is merged.

**Review by tbouffard — APPROVED** [review]
LGTM, I have a question about the fragment availability, see the related comment

File: modules/ROOT/pages/software-extensibility.adoc (hunk 13 above)
Comment (tbouffard) [reply modules/ROOT/pages/software-extensibility.adoc:65]: ping @vbotnari12

**Pull request merged**